<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDlg</class>
 <widget class="QDialog" name="PasswordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PasswordDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="2">
    <widget class="QPushButton" name="pushOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="6">
    <widget class="QLabel" name="labelMovie">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="linePsw3">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLabel" name="labelPsw3">
     <property name="text">
      <string>Re-enter new password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="linePsw2">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="labelPsw2">
     <property name="text">
      <string>Re-enter password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="linePsw1">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="labelPsw1">
     <property name="text">
      <string>Enter password</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>linePsw1</tabstop>
  <tabstop>linePsw2</tabstop>
  <tabstop>linePsw3</tabstop>
  <tabstop>pushOk</tabstop>
  <tabstop>pushCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
